src/app/config/or_options_st.h

```diff
@@ -634,6 +634,12 @@ struct or_options_t {
   /** If true, include statistics file contents in extra-info documents. */
   int ExtraInfoStatistics;
 
+  /** If true, activate reporting of all bandwidth statistics.
+   *  Defaults to auto which let's the consensus decide the reporting part.
+   *  If it is true and the consenus decides otherwise than the statistics
+   *  are written to disks but are not reported. */
```

Comment:
I'm sorry, this description doesn't match what the function does. If the BandwidthStatistics option is true, then the statistics are collected and reported.

Please use another consensus parameter option comment as a template.